src/main/resources/anniv5Resources/localization/eng/UIstrings.json

```diff
@@ -83,6 +83,19 @@
       "Watcher Veil"
     ]
   },
+  "${ModID}:BulwarkPack": {
+    "TEXT": [
+      "Bulwark",
+      "The greatest defensive tools of the Spire, together for the first time in one place.",
+      "modargo"
+    ]
+  },
+  "${ModID}:BulwarkPackHat": {
+    "TEXT": [
+      "TBD",
```

Comment:
Not sure how we've filled in hats for new packs like this, this is the one thing missing from the pack.